packages/dashboard/src/components/account-button-toggle/accounts-button-toggle.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { localize } from '@deriv/translations';
+import { ButtonToggle } from '@deriv/components';
+
+type TAccountsButtonToggleProps = {
+    onChangeAccount: (item: string) => void;
+    className?: string;
+};
+
+const AccountsButtonToggle: React.FC<TAccountsButtonToggleProps> = ({ onChangeAccount, className }) => {
```

Comment:
is possible to pass in `{...props}` to `<ButtonToggle />`? Something like `id` might be passed for e2e tests, etc.